### DISCUSSION TITLE
Socket IO polling

### DISCUSSION THREAD

**Discussion by FutureFutureTo** [resend/react-email#1748] [General]
I have a new build on a NextJS app.

I followed the Manual Install process for React Email. All seems good, but now I'm getting the following console when running the Next app in dev:

GET /socket.io?EIO=4&transport=polling

It's a constant request and is causing rate limiting in my parent app. Any ideas on how to kill this? Also, what is socket.io used for in this package?

**Comment by FutureFutureTo** [comment]
Whoops, solved it!

If anyone else is running into this, make sure you don't have a browser open with the React Email dashboard running.

**Reply by stvenchg** [reply]
Thanks! Had the same issue and it was exactly this.
